containerz/containerz.proto

```diff
@@ -144,6 +162,9 @@ message ImageTransfer {
   // Optional. Instructs the target to fetch the image from a remote location.
   // The above name and tag must be applied to the container once downloaded.
   common.RemoteDownload remote_download = 4;
+
+  // Indicates that the file to be transferred is a plugin.
+  bool is_plugin = 5;
```

Comment:
Should this be within a `oneof` or alternatively an enum? It seems like we might have some other types that are mutually exclusive that might be transferred via `Deploy`?

Reply:
That's not clear to me - I don't think there are other common types images. Technically, this only tells the server not to attempt to load the received image so perhaps it should be `no_load` instead of `is_plugin`